android/src/main/java/com/pspdfkit/react/ReactPdfViewManager.java

```diff
@@ -114,6 +148,44 @@ public void receiveCommand(PdfView root, int commandId, @Nullable ReadableArray
             case COMMAND_SAVE_CURRENT_DOCUMENT:
                 root.saveCurrentDocument();
                 break;
+            case COMMAND_GET_ANNOTATIONS:
+                if (args != null) {
+                    final int requestId = args.getInt(0);
+                    Disposable annotationDisposable = root.getAnnotations(args.getInt(1), args.getString(2))
+                            .subscribeOn(Schedulers.io())
+                            .observeOn(AndroidSchedulers.mainThread())
```

Comment:
Does the return message in RN require to be dispatched on the `mainThread()`? If not, it might make sense to drop this.